src/web/utils.ts

```diff
@@ -227,3 +232,34 @@ function spherical2tilt(altitudeAngle: number, azimuthAngle: number) {
 
   return { tiltX, tiltY };
 }
+
+const SVGElements = [
+  'Circle',
+  'ClipPath',
+  'Ellipse',
+  'ForeignObject',
+  'G',
+  'Image',
+  'Line',
+  'Marker',
+  'Mask',
+  'Path',
+  'Pattern',
+  'Polygon',
+  'Polyline',
+  'Rect',
+  'Svg',
+  'Symbol',
+  'TSpan',
+  'Text',
+  'TextPath',
+  'Use',
+];
+
+export function isSVGElement(viewRef: SVGRef | GestureHandlerRef) {
+  const className = Object.getPrototypeOf(viewRef).constructor.name;
+
+  return (
+    SVGElements.indexOf(className) >= 0 && Object.hasOwn(viewRef, 'elementRef')
+  );
```

Comment:
This is confusing. Where does `SVGRef` come from? Why do we check for `elementRef`?

Reply:
Commentary added in [81e85ac](https://github.com/software-mansion/react-native-gesture-handler/pull/3197/commits/81e85ac5ba8769a364c2ac50613587c76829e856)

Reply:
It might be helpful to prefix these checks with `RN` for `RNSVG` to avoid potential confusion, as it’s not part of the standard web API.

Reply:
> SVGRef is a type added in interfaces.ts file

That I can see, but where does `elementRef` come from?

Reply:
For future context - it's this https://github.com/software-mansion/react-native-svg/blob/38186e82833d7a7d6bb3efdf272b8919028e7df0/src/web/WebShape.ts#L23-L24